split-the-stream/README.md

```diff
@@ -0,0 +1,89 @@
+# Ideas about splitting the source-git work and upstream work
+
+Some ideas, possibilities, pros and cons of moving out the source-git related work.
+
+## What does the source-git workflow mean?
+
+Must have:
+
+- If user creates a merge-request on the source-git repository:
+  - Create a matching merge-request to the dist-git repository.
+  - Sync the CI results from the dist-git merge-request to the source-git merge-request.
+- If the dist-git is updated, update the source-git repository.
+
+Should have:
+
+- If the source-git merge-request is updated, update the dist-git merge-request.
+
+Could have:
+
+- User is able to re-trigger the dist-git CI from the source-git merge-request.
```

Comment:
@csomh @TomasTomecek Can you please verify this part so I can continue with the spike?

Reply:
Done.

Reply:
Thanks.